### DISCUSSION TITLE
training ASReview with one relevant paper for each exposure

### DISCUSSION THREAD

**Discussion by LukLuk-sudo** [asreview/asreview#1939] [Q&A]
Hi Everybody! can you give me a feedback on this method I decided to apply? It is not the SAFE method; and I am now wondering if it still good enough. 

 I am doing a systematic review with many different exposures (not necessary related one to another). 
To train the model we will start with at least 2 papers (one relevant and one not relevant) for each exposure... (I have 12 exposures, so 12 relevant)  Screening will be stopped after 20% of the records have been screened and 200 consecutive records have been labeled as irrelevant. To enhance the retrieval of “hard-to find papers” the second model will be used to re-arrange remaining records and then we will stop after 100 not relevant.    MAny thanks !

**Comment by FelixWdm** [comment]
Hi @LukLuk-sudo , this looks OK, but all depends on the total number of items in the set and the percentage of relevant papers in this set. And that depends on the search query off course and may differ per exposure. There's no ideal method, but thinking ahead like you've done now is already a great start. I hope this helps.

**Reply by LukLuk-sudo** [reply]
Hi @FelixWdm, Many thanks! I appreciate your insights, but I’m not entirely sure I understand what you meant about it depending on the number of items and the percentage of relevant papers. Since I don’t have that information prior to the screening, could you clarify a bit more what you had in mind? Were you referring to the total number of hits retrieved from the search string? (It’s more than 40,000.)

Also, considering the broad range of exposures I’m dealing with, do you think a different approach might yield better results? I’d love to hear any suggestions on potential risks I might be taking with my current approach and how you would recommend tackling them.

Thanks again :)
